Client/Frontend/Popup/PaymentHandlerPopupView.swift

```diff
@@ -0,0 +1,284 @@
+        defaultDismissType = .noAnimation
+        presentsOverWindow = true
+        
+        containerView = UIView()
+        containerView.autoresizingMask = [.flexibleWidth]
+        
+        if let imageView = imageView {
+            containerView.addSubview(imageView)
+            dialogImage = imageView
+        }
+        
+        titleLabel = UILabel()
+        titleLabel.textColor = BraveUX.greyJ
+        titleLabel.textAlignment = .center
+        titleLabel.font = UIFont.systemFont(ofSize: titleSize, weight: titleWeight)
+        titleLabel.text = title
+        titleLabel.numberOfLines = 0
+        containerView.addSubview(titleLabel)
+        
+        messageLabel = UILabel()
+        messageLabel.textColor = BraveUX.greyH
+        messageLabel.textAlignment = .center
+        messageLabel.font = UIFont.systemFont(ofSize: 15)
+        messageLabel.text = message
+        messageLabel.numberOfLines = 0
+        containerView.addSubview(messageLabel)
+        
+        displayItemTitle = UILabel()
+        displayItemTitle.textColor = BraveUX.greyH
+        displayItemTitle.textAlignment = .left
+        displayItemTitle.font = UIFont.systemFont(ofSize: 13, weight: UIFont.Weight.bold)
+        displayItemTitle.text = Strings.orderSummary
+        displayItemTitle.numberOfLines = 0
+        containerView.addSubview(displayItemTitle)
+        
+        totalItemTitle = UILabel()
+        totalItemTitle.textColor = BraveUX.greyH
+        totalItemTitle.textAlignment = .left
+        totalItemTitle.font = UIFont.systemFont(ofSize: 13, weight: UIFont.Weight.bold)
+        totalItemTitle.text = Strings.total
+        totalItemTitle.numberOfLines = 0
+        containerView.addSubview(totalItemTitle)
+        
+        addTotalLabel(message: "")
+        
+        if let inputType = inputType {
+            textField = UITextField().then {
+                $0.keyboardType = inputType
+                $0.textColor = .black
+                $0.placeholder = inputPlaceholder ?? ""
+                $0.autocorrectionType = .no
+                $0.autocapitalizationType = .none
+                $0.layer.cornerRadius = 4
+                $0.layer.borderColor = UIColor(white: 0, alpha: 0.3).cgColor
+                $0.layer.borderWidth = 1
+                $0.delegate = self
+                $0.textAlignment = .center
+                $0.isSecureTextEntry = secureInput
+                containerView.addSubview($0)
+            }
+        }
+        
+        updateSubviews()
+        
+        setPopupContentView(view: containerView)
+        setStyle(popupStyle: .dialog)
+        setDialogColor(color: BraveUX.popupDialogColorLight)
+    }
+            
+    func addDisplayItemLabel(message: String) {
+        let displayItem = UILabel()
+        displayItem.textColor = BraveUX.greyH
+        displayItem.textAlignment = .left
+        displayItem.font = UIFont.systemFont(ofSize: 12)
+        displayItem.text = message
```

Comment:
in all these functions, does setting `.text` automatically sanitize HTML characters like `<>`?

Reply:
`PaymentHandlerPopupView` will be removed completely. The UI work is in progress. 
https://github.com/brave/brave-ios/issues/2383. This work was done to understand how the payment modal will look like. I'll add a comment to this file.

I'll make a note of sanitizing the content when the PR for the final UI is available.

Reply:
https://github.com/brave/brave-ios/pull/2327/files#diff-9c93c71942418dd952c3647445166ad1R10